hack/build.sh

```diff
@@ -45,6 +45,8 @@ if [[ "${GOOS}" = "windows" ]]; then
   OUTPUT="${OUTPUT}.exe"
 fi
 
+go mod download
```

Comment:
is there a specific reason to call this here?

Reply:
done!